sound/soc/sof/sof-audio.h

```diff
@@ -279,6 +279,16 @@ struct snd_sof_pcm_stream {
 	 * active or not while suspending the stream
 	 */
 	bool suspend_ignored;
+
+	struct delayed_work wclk_calibration;
+	u64 stream_start_wclk; /* wclk in dsp */
+	u64 stream_start_offset;
+	struct sof_ipc4_fw_registers *fw_reg;
+	u32 llp_offset;
+	u32 accuracy; /* wclk resolution in ns */
```

Comment:
These are all some IPC4 specific variables and terms. I, for example have no idea what `stream_start_offset` might be and what unit it is specified, could be bytes, samples, ticks, time, length...
llp_offset, while I recall seeing similar thing, I still can not really figure out what it is.
The stream_start_wclk is likely a wclk tick when the stream is started?

These do not belong in here, imho. strictly IPC4 information to implement an IPC4 specific feature based on IPC4 only information.

Reply:
refined, thanks